source_gen/test/constants_test.dart

```diff
@@ -2,9 +2,7 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'package:analyzer/dart/constant/value.dart';
```

Comment:
any clue what this was used for?

Reply:
There was an explicit generic type parameter on line 127 that is no longer necessary because of better type inference

Reply:
sweet!